tests/integration/component.go

```diff
@@ -404,14 +407,12 @@ func componentTests(args ...string) {
 			It("should be valid if path is relative and includes ../", func() {
 				relativeContext := fmt.Sprintf("..%c%s", filepath.Separator, filepath.Base(commonVar.Context))
 				fmt.Printf("relativeContext = %#v\n", relativeContext)
```

Comment:
Do we need the print statement here?

Reply:
I guess we don't need it, I guess it was added long back by Tomas.

Reply:
I will remove and push it.